src/data/markdown/docs/02 javascript api/07 k6-experimental/03 websockets/10 WebSocket/30-WebSocket-onclose.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'WebSocket.onclose'
+excerpt: 'A handler function for WebSocket connection close event.'
+---
+
+A handler for a WebSocket connection `close` event.
+For multiple, simultaneous event handlers, use [`WebSocket.addEventListener()`](/javascript-api/k6-experimental/websockets/websocket/websocket-addeventlistener).
+
+| Parameter | Type     | Description                                  |
+| --------- | -------- | -------------------------------------------- |
+| handler  | function | The function to call when the event happens. |
```

Comment:
I think we can drop this table for all handler functions, unless the function itself takes a parameter (like `onmessage`), in which case we would describe its parameters.

Reply:
I agree :+1: 

For that PR, let's maybe drop all such tables since they don't bring any additional value. And for the `onmessage` (and similar handlers) let's document the handler's function signature in the following PRs.